### PR TITLE
Modified WebDriverFactory.jsExecute() method to return value

### PR DESCRIPTION
**Pull request by dexguitar**

### CODE REVIEW

File: jdi-light-bdd-tests/src/test/resources/features/WebPage.feature [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@WebPage
```
Comment (ChepelVE) [inline jdi-light-bdd-tests/src/test/resources/features/WebPage.feature]: - @WebPage from lower case

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/WebPageSteps.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
+package com.epam.jdi.bdd.stepdefs;
```
Comment (ChepelVE) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/WebPageSteps.java]: - add start/end line anchor (^/$ respectively)
- add (?:I |) at the beginning of action steps

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java [hunk 3]
```diff
@@ -126,6 +122,17 @@ public void setAttribute(String name, String attrName, String attrValue) {
 		BaseUIElement el = getUI(name);
 		el.setAttribute(attrName, attrValue);
 	}
+
+	@When("I go forward")
```
Comment (ChepelVE) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java]: - add start/end line anchor (^/$ respectively)
- add (?:I |) at the beginning of action steps

File: jdi-light-bdd-tests/src/test/resources/features/WebPage.feature (hunk 1 above)
Comment (dexguitar) [reply jdi-light-bdd-tests/src/test/resources/features/WebPage.feature]: Resolved

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/WebPageSteps.java (hunk 2 above)
Comment (dexguitar) [reply jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/WebPageSteps.java]: Resolved

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java (hunk 3 above)
Comment (dexguitar) [reply jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java]: Resolved

**Pull request merged**